checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/views/AbstractCardFieldLengthFilter.kt

```diff
@@ -10,6 +10,11 @@ import com.worldpay.access.checkout.validation.CardBrandUtils
 import com.worldpay.access.checkout.validation.CardValidator
 import com.worldpay.access.checkout.validation.ValidationResult
 
+/**
+ * [AbstractCardFieldLengthFilter] which is used to restrict length of a card field
```

Comment:
[AbstractCardFieldLengthFilter] is a common abstraction class which is used by individual field implementers to restrict the length of a particular card field